src/hooks/useSlippageToleranceFromTrade.ts

```diff
@@ -29,9 +28,13 @@ function guesstimateGas(trade: Trade<Currency, Currency, TradeType> | undefined)
 const MIN_AUTO_SLIPPAGE_TOLERANCE = new Percent(5, 1000) // 0.5%
 const MAX_AUTO_SLIPPAGE_TOLERANCE = new Percent(25, 100) // 25%
 
-export default function useSwapSlippageTolerance(
+/**
+ * @param trade
+ * @returns slippage tolerance based on values from current trade, gas estimates from api, and active network
```

Comment:
Type the return value to make it more obvious (and catch issues if you return the wrong thing).
Feel free to just write "Returns ..." instead of using the jsdoc keywords - this is TypeScript, so that information is captured via types anyways.